modules/network/kademlia/kademlia.js

```diff
@@ -431,6 +413,38 @@ class Kademlia {
             }
         });
 
+        this.node.use('*', (request, response, next) => {
+            if (request.params.header) {
+                const header = JSON.parse(request.params.header);
+                const destContact = header.to;
+                header.ttl -= 1;
+                if (header.ttl > 0) {
+                    this.log.info(`Request received for ${destContact}`);
```

Comment:
Maybe to put here lower logging lever (trace or debug)?
Also change console.log on lilne 429